### PR TITLE
[stable/sonarqube] Adding support for Service annotations

### PR DESCRIPTION
**Pull request by marcostvz**
I needed to add annotations to service to allow an internal load balancer in GCP. I guess this could be useful for others.

Inspired in https://github.com/kubernetes/charts/blob/master/stable/kong/templates/service-kong-admin.yaml

### CODE REVIEW

**Comment by unguiculus** [comment]
cc @rjkernick

**Comment by shashiranjan84** [comment]
@marcostvz please try https://github.com/kubernetes/charts/pull/3947, it has more flatten value file.

**Comment by unguiculus** [comment]
Please add documentation.

**Comment by marcostvz** [comment]
Hey @unguiculus tell me if is enough adding that to README.md

**Comment by unguiculus** [comment]
Please also add an empty default to `values.yaml`.

**Comment by marcostvz** [comment]
Done, thanks for your comment

File: stable/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.3.6
+version: 0.3.7
```
Comment (timstoop) [inline stable/sonarqube/Chart.yaml]: This adds a feature, so please bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]

**Comment by timstoop** [comment]
Making that change will retest and hopefullt fix the container not ready error.

**Comment by rjkernick** [comment]
Please rebase off of master. The tests should pass with the fixes that are on master.

**Comment by marcostvz** [comment]
Thanks @rjkernick , it looks good now 🎉

**Pull request merged**